_posts/2019-05-23-assessing-rails-upgrades.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Is this Rails Upgrade Project a good fit?"
+date: 2019-05-23 15:00:00
+categories: ["rails", "upgrades"]
+author: "emily"
+---
+
+Over the years we have seen a significant increase in the  demand for [Ruby on Rails](http://rubyonrails.org/) upgrades. That's why at [Ombu Labs](https://www.ombulabs.com) we decided to develop [FastRuby.io](https://fastruby.io), our Ruby on Rails upgrade service.
+
+When we receive an inquiry via [FastRuby.io](https://fastruby.io), **the first thing we like to do is schedule a call with the potential client to get more details about their app**.
+
+<!--more-->
+
+In the call we like to get the following info:
+
+1. A general overview of their app and why they are interested in upgrading Rails
+2. What **Rails version is their app currently on**?
+3. What **Rails version do they want to upgrade to**?
+4. What **Ruby version** is their app using?
+5. How large is their app? How many lines of code does it have?
+6. **How much test coverage does their app have**?
+7. Are there **any major blockers** that they know of? Have they tried to upgrade before?
+8. Do they have any **time limitations or deadlines**? How fast do they want to start working on the upgrade?
+
+These questions help us get an idea of the scope and level of risk of the Rails upgrade. Asking the potential client about the test coverage on their app is crucial to assess risk. **We generally don’t upgrade Rails apps that have less than 60% test coverage**. Similarly, asking about the potential client’s timeframe and deadlines can be helpful to determine if we have the availability and bandwith to complete the Rails upgrade.
+
+Based on the information from these initial [FastRuby.io](https://fastruby.io) calls, we can determine whether or not the Rails upgrade project is a good fit for the team.
+
+If you don't have the time to upgrade your Rails app, check out [FastRuby.io](https://fastruby.io) and schedule a call with us.
```

Comment:
You could add a note about Rails 6.0 being just around the corner, and maybe link to this article? https://fastruby.io/blog/rails/upgrades/upgrade-rails-from-5-2-to-6-0.html